README.md

```diff
@@ -15,6 +15,8 @@
 <!-- A spacer -->
 <div>&nbsp;</div>
 
+Master ![CI](https://github.com/gleam-lang/gleam/workflows/CI/badge.svg?branch=master)
```

Comment:
Could you put this in line with the other badges please? And remove the word "master". Thanks!